main.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'classes/person'
+require_relative 'classes/capitalize_decorator'
+require_relative 'classes/trimmer_decorator'
+
+person = Person.new(22, 'maximilianus')
+puts person.correct_name
+
+capitalized_person = CapitalizeDecorator.new(person)
+puts capitalized_person.correct_name
+
+capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
+puts capitalized_trimmed_person.correct_name
```

Comment:
- [ ] Great job so far 👏🏻 However the  TrimmerDecorator class should check if the name has a maximum of 10 characters and trim the word if it's longer than 10. Here is the screenshot below 👇🏻 the third line should trim the work instead, it returns the full name `Maximilianus`. Kindly correct this 👍🏻 

![image](https://user-images.githubusercontent.com/79058364/223983543-3c9d6149-882b-4849-9245-26d339ade734.png)

Reply:
I suggest you use the tenray operator for this 👇🏾 

```ruby
@nameable.correct_name.size > 10  ? @nameable.correct_name[0, 10] : @nameable.correct_name
```

Reply:
No way! It can't be... now I missed the other condition... 🤣 Well, I think you two are right, this is a perfect case for the ternary operator, Thank you!